source/data-api/authenticate.txt

```diff
@@ -67,6 +67,15 @@ For example, the following request uses Bearer Authentication:
         }
       }'
 
+In general, bearer authentication with an access token has higher
```

Comment:
seems you're using the exact text in this paragraph 4 times. should it just be an include?